Test/sampleModFileDouble2.txt

```diff
@@ -3,5 +3,6 @@ TG   Nxs or Nxt
 PP   Anywhere.
 MM   15.995
 NL   0 or 15
+OM   343
```

Comment:
Yea, it is different because modifications observed on C have a histogram peak at a value different than MM, due to the fixed modification. Now I'm thinking this should not be a property of a modification, but rather handled in the gptmd code in metamorpheus. I'll remove the OM field completely. What do you think?